### PR TITLE
Add/extend cli commands

### PR DESCRIPTION
**Pull request by Yawning**
* `inspect governance-proposal`
 * `inspect runtime-stats`
 * `inspect node-status`
 * `inspect native-token`
 * `network add-local`
 * `accounts burn`
 * `accounts from-public-key`
 * `accounts info` (display more information)

### CODE REVIEW

File: cli/cmd/accounts.go [hunk 1]
```diff
@@ -57,33 +59,126 @@ var (
...
+			sa := consensusAccount.Escrow.StakeAccumulator
+			if len(sa.Claims) > 0 {
+				fmt.Println("  Stake Accumulator:")
+				sa.PrettyPrint(ctx, "    ", os.Stdout)
+				fmt.Println()
+			}
+
+			if npw.ParaTime != nil {
+				// I'm too smoothbrain to figure out how to convert from
+				// height to round, instead of complaining about this in
```
Comment (kostko) [inline cli/cmd/accounts.go:158]: Something like:
```golang
blk, err := c.Consensus().RootHash().GetLatestBlock(ctx, &RuntimeRequest{
    RuntimeID: npw.ParaTime.Namespace(),
    Height:    common.GetHeight(),
})
cobra.CheckErr(err)
round := blk.Header.Round

File: cli/cmd/network.go [hunk 2]
```diff
@@ -69,35 +72,66 @@ var (
...
+	networkAddLocalCmd = &cobra.Command{
+		Use:   "add-local <name> <rpc-endpoint>",
+		Short: "Add a new local network",
+		Args:  cobra.ExactArgs(2),
+		Run: func(cmd *cobra.Command, args []string) {
+			cfg := cliConfig.Global()
+			name, rpc := args[0], args[1]
+
+			net := config.Network{
+				// ChainContext: chainContext,
```
Comment (kostko) [inline cli/cmd/network.go:94]: Remove?

**Review by kostko — APPROVED** [review]

File: cli/cmd/accounts.go (hunk 1 above)
Comment (Yawning) [reply cli/cmd/accounts.go]: `Error: rpc error: code = Unimplemented desc = unknown service oasis-core.RuntimeClient`

or

`Error: rpc error: code = Unimplemented desc = Only some methods are allowed on this proxy.`

Comment (Yawning) [reply cli/cmd/accounts.go]: Oh well.

**Pull request merged**